src/autostep/NewProjectCommand.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.CommandLine;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace AutoStep.CommandLine
+{
+    public class NewProjectCommand : AutoStepCommand<RunArgs>
```

Comment:
As a rule, there should only be one top-level class per-file (among other things, makes it easier to review). Can you split the file out?

Reply:
Will do 👍